src/site/content/en/mini-apps/what-are-h5-and-quickapp/index.md

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
Not sure how we could title this URL while preserving the `mini-app` prefix

Reply:
Done.